src/streamlink_cli/argparser.py

```diff
@@ -467,6 +467,15 @@ def boolean(value):
 
 stream = parser.add_argument_group("Stream options")
 stream.add_argument(
+    "--url",
+    dest="url_param",
+    metavar="URL",
+    help="""
+    Alternative parameter to specify a URL to attempt to
```

Comment:
I think it's good to have the help section there, just wondering if there's anything we can do to make it more clear (though I don't have a clarity suggestion so I can't really talk). I'm thinking we should probably start a doc page that specifically explains things like the configuration file. Maybe a more in-depth getting started guide or something.